src/base/App.js

```diff
@@ -1,18 +1,21 @@
 //Dependencies
 import React from 'react';
-import { connect } from 'react-redux';
+import { connect, useDispatch } from 'react-redux';
```

Comment:
I think we can refactor this to not use connect right? instead you should be able to use hooks